src/js/components/FileInput/FileInput.js

```diff
@@ -25,7 +25,7 @@ const ContentsBox = styled(Box)`
   position: relative;
   ${props => (props.disabled ? disabledStyle() : 'cursor: pointer;')}
   &:focus {
-    ${focusStyle}
+    outline: none;
```

Comment:
would this be a candidate for `unFocusStyle`  from utils?

Reply:
good question I didn't know that exists let me check it out!